ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/core/StandardComponentInitializer.java

```diff
@@ -497,7 +498,11 @@ public void initialize() {
         loaderToolbarGroup.add(loaderAction);
 
         // Define hot-keys
-        keyBinding.getGlobal().addKey(new KeyBuilder().action().alt().charCode('n').build(), "navigateToFile");
+        if(UserAgent.isMac()){
```

Comment:
BTY I think it would be better to write
```
keyBinding.getGlobal().addKey(new KeyBuilder().action().alt().charCode(UserAgent.isMac()? 'N' : 'n').build(), "navigateToFile");
```